relay-sampling/src/lib.rs

```diff
@@ -365,6 +365,10 @@ impl FieldValueProvider for Event {
                 Value::Bool(relay_filter::browser_extensions::matches(self))
             }
             "event.web_crawlers" => Value::Bool(relay_filter::web_crawlers::matches(self)),
+            "event.transaction" => match self.transaction.0 {
```

Comment:
I don't understand why you are doing those conversions, you can use the methods provided by the ToValue trait for this.

Reply:
`IntoValue` converts into our `relay_general::Value` type , this is a `serde_json::Value` ... are you suggesting using the `realy_geneeral::Value` type ?

Reply:
IntoValue::??? and then `Into::into` to get from our `Value` to serde's `Value`... duked out IRL, your way is more readable albeit verbose